src/main/java/org/gridsuite/network/map/dto/definition/tieline/TieLineMapInfos.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.dto.definition.tieline;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.network.map.dto.ElementInfos;
+import org.gridsuite.network.map.model.CurrentLimitsData;
+
+/**
+ * @author Rehili Ghazwa <ghazwa.rehili at rte-france.com>
+ */
+@SuperBuilder
+@Getter
+public class TieLineMapInfos extends ElementInfos {
+
+    private String voltageLevelId1;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private String voltageLevelName1;
+
+    private String voltageLevelId2;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private String voltageLevelName2;
+
+    private Boolean terminal1Connected;
+
+    private Boolean terminal2Connected;
```

Comment:
I have seen that for hvdcs all those values are `@JsonInclude(JsonInclude.Include.NON_NULL)`

Is it different for tie lines ? Their voltage levels may have no ids ?

(in toMapInfos later your name may be null, but not the id....?)